### PR TITLE
[SDK Renaming] - New Carthage Version & Adjustments

### PR DESCRIPTION
**Pull request by AlejandroOrozco**
## Jira Ticket
- ACCSEC-

## Github Issue
- ISSUE-

## Description
- Update Readme instructions for Carthage installation using the latest release `0.38.0``
- Update CircleCI & Fastlane flow to zip the framework as `TwilioVerifySDK.framework.zip` to complain the requirements of Carthage.  
 

## Commit message
-  

## Testing
- [ ] Added unit tests
- [x] Ran unit tests successfully
- [x] Added documentation for public APIs and/or Wiki

### CODE REVIEW

**Comment by AuthyApps** [comment]
## Code coverage
### Total coverage: **`95.13%`**
> Powered by [Slather](https://github.com/SlatherOrg/slather)

<p align="right" data-meta="generated_by_UnitTests">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by AuthyApps** [comment]
## Code coverage
### Total coverage: **`95.18%`**
> Powered by [Slather](https://github.com/SlatherOrg/slather)

<p align="right" data-meta="generated_by_UnitTests">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: README.md [hunk 1]
```diff
@@ -71,16 +71,7 @@ pod 'TwilioVerify', '~> 0.4.0'
...
-    - This will require using the script to strip the framework's architecture before lipo command fails: https://github.com/Carthage/Carthage/blob/master/Documentation/Xcode12Workaround.md#workaround-script
-    - Also make sure to exclude the **arm64** architectures from your targets at **Built Settings** > **Architectures** > Exclude from **Any iOS Simulator SDK** > **arm64**
+Since version `1.0.0` of `TwilioVerifySDK` the prebuilt asset fat version `.framework` is been deprecated, to give space for the universal framework `.xcframework`. Make sure to use the new version of Carthage [0.38.0](https://github.com/Carthage/Carthage/pull/3152) that was release in order to support the `xcframework` assets, by using this version or a superior one, Carthage will download and unzip the `TwilioVerifySDK.framework.zip` attached in the release version, resulting in a `TwilioVerifySDK.xcframework` that can be found in the build folder of Carthage.
```
[the remark this answers is not in the document]
Comment (AlejandroOrozco) [reply README.md]: That makes sense, Thank you. I just updated that reference 👍 .

**Comment by AuthyApps** [comment]
## Code coverage
### Total coverage: **`95.13%`**
> Powered by [Slather](https://github.com/SlatherOrg/slather)

<p align="right" data-meta="generated_by_UnitTests">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Review by SergioFierro — APPROVED** [review]

**Review by yafuquen — APPROVED** [review]

**Pull request merged**